packages/components/src/components/breadcrumb-item/breadcrumb-item.custom.styles.ts

```diff
@@ -0,0 +1,70 @@
+import { css } from 'lit';
+
+export default css`
+  .breadcrumb-item {
+    color: var(--syn-color-neutral-500);
+    font-size: var(--syn-font-size-x-small);
+    font-weight: var(--syn-font-weight-normal);
+  }
+
+  /**
+   * Make sure to remove the border radius.
+   * Without this, focus will be rounded
+   */
+  .breadcrumb-item__label {
+    border-radius: var(--syn-border-radius-none);
+  }
+
+  :host(:not(:last-of-type)) .breadcrumb-item__label {
+    color: var(--syn-typography-color-text);
+  }
+
+  :host(:not(:last-of-type)) .breadcrumb-item__label:hover {
+    color: var(--syn-color-primary-600);
+  }
+
+  :host(:not(:last-of-type)) .breadcrumb-item__label:active {
+    color: var(--syn-color-primary-950);
```

Comment:
The `active` styling / color can't be differentiated that well vs the normal styling / color: 
See left normal and right active:
![image](https://github.com/synergy-design-system/synergy-design-system/assets/115139104/f4c83d02-e28d-4eab-9b8a-ce4132b60c9e)

In the current figma version the `active` color is neutral-500

Reply:
I think this is not the actually `:active` pseuso-state, but the active element (e.g. the last one). @LeaLeRose: Could you give some information here?

Reply:
I think you are right.

But I think we should add an :active state for pressing down a breadcrumb-item, which is more visible. Especially after the statement of our stakeholder, who said he misses a visual pressed state for some components. Or we postpone this topic to a later time and we add the pressed / active state for all components simultaniously.

Reply:
The active one is the last item in the breadcrumb list, so it works as intended